Tests/Publish-ProGetUniversalPackage.Tests.ps1

```diff
@@ -0,0 +1,144 @@
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-Tests.ps1' -Resolve)
+
+
+$packagePath = '.\WhsUniversalPackageTest-0.1.1.upack'

```

Comment:
Remove the "Whs" portion to these names. Since we plan on releasing this as open-source software, we don't want any references to WebMD in the code.